components/partials/activity/positions/positions.vue

```diff
@@ -215,7 +214,6 @@ export default Vue.extend({
     pollOrderbooks() {
       this.poll = setInterval(() => {
         this.$accessor.positions.fetchSubaccountPositions() // refresh mark price
```

Comment:
Here as well, please rename this functionality to something else so it's clear what it does, even if it means to duplicate the `fetchSubaccountPositions` function.